### PR TITLE
Update pedsnet-etlprov usage in README.md

### PR DESCRIPTION
**Pull request by murphyke**

### CODE REVIEW

File: cmd/etlprov/README.md [hunk 1]
```diff
@@ -3,9 +3,18 @@
 ## Usage
 
 ```bash
-$ pedsnet-etlprov [-model <model>] [-version <version>] [-service <service>] <dir>
+$ pedsnet-etlprov [-model <model>] [-version <version>] [-truncate false] [-ignore <entities>] [-service <service>] [-ver] <dir>
```
Comment (l-k-) [inline cmd/etlprov/README.md]: I think it should be `[-truncate=false]`. At least it doesn't work without the "=" right now.

Comment (l-k-) [inline cmd/etlprov/README.md]: Also, [-v], not [-ver]

File: cmd/etlprov/README.md [hunk 2]
```diff
@@ -3,9 +3,18 @@
...
 ```
 
+Option | Description
+---|---
+-model <model> | Specify a data model other than 'pedsnet', e.g. 'i2b2_pedsnet'
+-version <version> | Specify PEDSnet CDM version, e.g. 2.0.0 or 2.1.0
+-truncate false | Show all errors, even redundant or excessive ones
+-service <service> | Specify model service other than http://data-models.origins.link (not useful unless you run your own model service)
+-ignore <entities> | Ignore a comma-separated list of entities (not normally used)
+-ver | Show the version of this tool
```
Comment (l-k-) [inline cmd/etlprov/README.md]: -ver -> -v

**Comment by murphyke** [comment]
@l-k- Thanks, as you can see, I was just making it up.  Will change.

**Comment by l-k-** [comment]
The new readme was very helpful actually!

**Comment by murphyke** [comment]
If you think it's accurate, you might as well merge it. I wanted to point it out to sites when I tell them about the validator download.

**Comment by l-k-** [comment]
Yep, it does look good to me -- but do you want to squash the two commits together, so that the commit history is cleaner?

**Comment by murphyke** [comment]
Not really, but I did it anyway and rebased against the latest master.

**Comment by l-k-** [comment]
Thanks -- for squashing, and for the honest reply :-D

**Pull request merged**